spec/features/signs_in.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+feature "signing in" do
+  scenario "with existing username and password" do
+    daniel = Fabricate(:user)
+    feature_sign_in(daniel)
+    page.should have_content(daniel.first_name)
```

Comment:
`should` syntax is no longer recommended by the creators of `rspec`, you should use the `expect` syntax:

```ruby
expect(page).to have_content(daniel.first_name)
```